release-doc-in-process/all_vf.TTL

```diff
@@ -346,9 +366,82 @@ vf:fulfills     a owl:ObjectProperty ;
 #        rdfs:range      xsd:boolean ;
 #        rdfs:comment    "The commitment or process is complete or not." .
 
-# isSubstitutable "Defines if any resources of that type can be freely substituted for any other resource of that type when used, consumed, traded, etc."
+vf:isSubstitutable a    owl:DatatypeProperty ;
+        rdfs:label      "is substitutable" ;
+        rdfs:domain     vf:ResourceSpecification ;
+        rdfs:range      xsd:boolean ;
+        rdfs:comment    "Defines if any resource of that type can be freely substituted for any other resource of that type when used, consumed, traded, etc." .
+
+vf:unit a               owl:ObjectProperty ;
+        rdfs:label      "unit" ;
+        rdfs:domain     vf:ResourceSpecification ;
+        rdfs:range      qudt:unit ;
+        rdfs:comment    "The unit expected for this resource specification as a default." .
+
+vf:definedQuantity a    owl:ObjectProperty ;
+        rdfs:label      "defined quantity" ;
+        rdfs:domain     vf:RecipeFlow ;
+        rdfs:range      qudt:QuantityValue ;
+        rdfs:comment    "The amount and unit defined for this recipe flow." .
+
+# TODO: need to define this, something like currency, inventory, skill, service; may be other aspects too.
+#vf:resourceCategory a    owl:ObjectProperty ;
+#        rdfs:label      "category" ;
+#        rdfs:domain     vf:ResourceSpecification ;
+#        rdfs:comment    "Defines the category of behavior expected for the type of resource." .
+
+vf:developedBy a        owl:ObjectProperty ;
+        rdfs:label      "developed by" ;
+        rdfs:domain     vf:Recipe ;
+        rdfs:range      foaf:Agent ;
+        rdfs:comment    "The agent who developed this recipe." .
+
+vf:variantOf a          owl:ObjectProperty ;
```

Comment:
I don't think this comes as fundamental and possibly we can take advantage of PROV https://www.w3.org/TR/prov-primer/

Reply:
I borrowed variantOf and successorOf from GR.  Although now I notice they have isVariantOf.  How would we use prov?

I think forking and versioning recipes is not fundamental to REA, but is an important part of supporting open hardware specifications.  I don't care too much how we do it.

Reply:
I've created dedicated issue for it https://github.com/valueflows/valueflows/issues/298

I would prefer to start with less and proposed `vf:variantOf` and `vf:successorOf` do not seem to me fundamental enough to try to get them in this first pass.

Reply:
OK, I will drop them from this PR.  Probably good to take a more comprehensive look.

Reply:
Done.